src/transformStateWithClones.js

```diff
@@ -65,7 +65,31 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
+  let stateClone = Object.assign({}, state);
```

Comment:
в тебе немає потреби заміняти посилання на клон тому const
```suggestion
  const stateClone = Object.assign({}, state);
```